anki/tags.py

```diff
@@ -102,7 +102,7 @@ def bulkAdd(self, ids, tags, add=True):
         res = self.col.db.all(
             "select id, tags from notes where id in %s and (%s)" % (
                 ids2str(ids), lim),
-            **dict([("_%d" % x, '%% %s %%' % y)
+            **dict([("_%d" % x, '%% %s %%' % y.replace('*', '%'))
                     for x, y in enumerate(newTags)]))
```

Comment:
I was deadly afraid of breaking this part, so I kept the change to an absolute minimum. I agree it has some readability issues.